src/Tempest/Console/src/Components/InteractiveComponentRenderer.php

```diff
@@ -153,6 +154,8 @@ private function applyKey(InteractiveConsoleComponent $component, Console $conso
                 continue;
             }
 
+            Fiber::suspend();
```

Comment:
What's this one about?

Reply:
This is further improved and commented in another upcoming PR because this class is complex, but it's needed to re-render components before they return a value.